android/build.gradle

```diff
@@ -1,37 +1,53 @@
 group 'com.pkmnapps.nearby_connections'
-version '1.0-SNAPSHOT'
+version '1.0'
 
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.2'
+        classpath 'com.android.tools.build:gradle:7.4.2'
     }
 }
 
 rootProject.allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```

Comment:
Any particular reason for changing this? It seens like jcenter is a superset of mavencentral

Reply:
https://developer.android.com/build/jcenter-migration